dispatch.go

```diff
@@ -64,6 +65,13 @@ func dispatch(ctx context.Context, pl *PairingLogic, cmd string, cmdArgs []strin
 			break
 		}
 
+		accessToken, err := pl.adb.GetKey(ctx, "rc-accesstoken", "key")
+		if err != nil {
+			log.Printf("Something weird happened trying to read the RC API access token from the database: %s", err)
+		}
+
+		rec.currentlyAtRC = pl.rcapi.userIsCurrentlyAtRC(accessToken, userEmail)
```

Comment:
This makes sure we store the proper `currentlyAtRC` value when a user first subscribes to pairing bot.